ci/Jenkinsfiles/release.groovy

```diff
@@ -33,7 +33,9 @@ pipeline {
   environment {
     BRANCH_NAME = "${params.BRANCH}"
     BUILD_VERSION = "${params.BUILD_VERSION}"
-    JIRA_NUXEO_ADDON_MOVING_VERSION = 'coldstorage-2023.x'
+    JIRA_PROJECT = 'NXP'
```

Comment:
needed?

Reply:
+ This makes it easier to have repository specific things in the same place and a generic _Release Project_ block.